release-notes/opensearch-release-notes-2.4.0.md

```diff
@@ -0,0 +1,704 @@
+* Mapping Logs/Rule fields to ECS(Elastic Common Schema) format ([#30](https://github.com/opensearch-project/security-analytics/pull/30), [#35](https://github.com/opensearch-project/security-analytics/pull/35), [#46](https://github.com/opensearch-project/security-analytics/pull/46), [#46](https://github.com/opensearch-project/security-analytics/pull/46), [#89](https://github.com/opensearch-project/security-analytics/pull/89))
+* Integrate Findings (Lifecycle Management including Rollovers), Triggers, Alerts(Lifecycle Management) ([#39](https://github.com/opensearch-project/security-analytics/pull/39), [#54](https://github.com/opensearch-project/security-analytics/pull/54), [#67](https://github.com/opensearch-project/security-analytics/pull/67), [#70](https://github.com/opensearch-project/security-analytics/pull/70), [#70](https://github.com/opensearch-project/security-analytics/pull/70), [#82](https://github.com/opensearch-project/security-analytics/pull/82))
+* Integrate with Notifications, Acknowledge Alerts ([#71](https://github.com/opensearch-project/security-analytics/pull/71), [#75](https://github.com/opensearch-project/security-analytics/pull/75), [#85](https://github.com/opensearch-project/security-analytics/pull/85))
+* Integrate with Security, implement RBAC, backend roles filtering ([#78](https://github.com/opensearch-project/security-analytics/pull/78))
+
+
+### OpenSearch Security Analytics Dashboards
+* Added create detectors UI pages. ([#15](https://github.com/opensearch-project/security-analytics-dashboards-plugin/pull/15)
+* Added get mappings API service wrappers. ([#22](https://github.com/opensearch-project/security-analytics-dashboards-plugin/pull/22))
+* Added overview page UI. ([#24](https://github.com/opensearch-project/security-analytics-dashboards-plugin/pull/24))
+* Added configure alerts UI page. ([#25](https://github.com/opensearch-project/security-analytics-dashboards-plugin/pull/25))
+* Added rules UI pages. ([#55](https://github.com/opensearch-project/security-analytics-dashboards-plugin/pull/55))
+* Added toast notifications. ([#70](https://github.com/opensearch-project/security-analytics-dashboards-plugin/pull/70))
+
+
+### OpenSearch Security Dashboards Plugin
+* Initial commit for multiple authentication ([#1110](https://github.com/opensearch-project/security-dashboards-plugin/pull/1110))
+* Saved Object Aggregation View ([#1146](https://github.com/opensearch-project/security-dashboards-plugin/pull/1146))
+* [Saved Object Aggregation View] Use namespace registry to add tenant filter ([#1169](https://github.com/opensearch-project/security-dashboards-plugin/pull/1169))
+* Move tenant-related utils to common folder ([#1184](https://github.com/opensearch-project/security-dashboards-plugin/pull/1184))
+
+
+### OpenSearch SQL
```

Comment:
Need confirmation from SQL team regarding of the features part. @dai-chen

Reply:
@zelinh I confirmed with Ani that all items in https://github.com/opensearch-project/sql/blob/2.4/release-notes/opensearch-sql.release-notes-2.4.0.0.md#features are not experimental and should be present in `features` section here. If you want to keep one line for each, you can use the subsection title as feature name. Thanks!